web/settings/instance.go

```diff
@@ -38,8 +39,13 @@ func getInstance(c echo.Context) error {
 	if err != nil {
 		return err
 	}
+	doc.Type = consts.Settings
 	doc.M["locale"] = instance.Locale
 
+	if err = permissions.Allow(c, permissions.GET, doc); err != nil {
```

Comment:
`getInstance` should also be accessible with the register token. Can you add this case?